### ISSUE TITLE
Danalock V3-BTZBE

### ISSUE THREAD

**Issue by danirebollo** [danielwelch/hassio-zigbee2mqtt#450]
**zigbee2mqtt add-on version (if edge, please report commit hash):**
1.15.0

**Operating environment (HassOS, Virtual Machine, Device/platform running Home Assistant):**
Rpi with Home Assistant connected through serial emulation to ESP8266+CC2530

**Description of problem:**
Danalock V3 BTZB is supported but not recognized in my installation ( https://www.zigbee2mqtt.io/devices/V3-BTZB.html ).
My submodel is BTZBE, so maybe this submodel it is not configured. The weird thing is that using SLS gateway (with ESP32 + CC2530: https://github.com/slsys/Gateway ) seems like zigbee2MQTT recognices manufacturer and model (although it is not working), and using Home assistant manufacturer and model are "unknown".

How I can recognize and use this lock?

Thanks

**Your logs from Home Assistant**
(MQTT subscribe logs)
```
br1/0xd0cf5efffe0391b2/set {"state":"UNLOCK"}
br1/bridge/logging {"level":"warning","message":"Device with modelID 'V3-BTZBE' is not supported."}
br1/bridge/logging {"level":"warning","message":"Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html"}
```

**Comment by ciotlosm** [comment]
@danirebollo I recommend you open an issue here: https://github.com/Koenkk/zigbee2mqtt/issues/new/choose
You can also try and patch it yourself by using https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/blob/dev/zigbee2mqtt/DOCS.md#using-external_converters however that is only fixed in this repo: https://github.com/zigbee2mqtt/hassio-zigbee2mqtt for now.

**Comment by Jopand** [comment]
Has this been looked at? I'm pretty sure it works with the same protocol as the old "V3-BTZB", that is supported. Where can I find these settings to make my own fix?

**Comment by dimadusk** [comment]
Great news :smiley: Zigbee was fixed with new Update version 0.22.4, DanaLock v3 (zigbee) works for more than 24 hours and didn’t loose connection since then. I could not use the lock after updating from firmware 0.17.4
![IMG_20231201_164355](https://github.com/danielwelch/hassio-zigbee2mqtt/assets/60470490/8a97f8df-e09f-4a42-bafa-6a6f8612df1b)
